src/_pytest/capture.py

```diff
@@ -495,6 +504,7 @@ def suspend_capturing(self, in_=False):
             self._in_suspended = True
 
     def resume_capturing(self):
+        self._suspended = False
```

Comment:
I noticed `stop_capturing` doesn't set `self._suspended = True` and I can see why, it is not really suspended after all, it has stopped. But that might mean we don't want a boolean, but a "state" string with `"capturing"`, `"suspended"` and `"stopped"`. What do you think?

Reply:
Makes sense.  Could also be set to `None` there though.

Reply:
Yep, but in that case `"stopped"` might be better for debugging purposes, and we can even start it with a proper string `"not initialized"` or something.

Reply:
Yeah.. note however that MultiCapture has `_reset` also already.
I suggest adding `_state` (string), and removing `_reset` then.

Reply:
Sounds good. 👍

Reply:
Done.